Grid-Stat/pre_processing_config.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+##################################################################################
+# Description
+##################################################################################
+# This configuration file is for the MET-tools workflow. These environmental 
+# variables are defined for the use of the pre-processing bash scripts in the 
+# workflow. Furthermore, in this file are a few Singularity container-dependent 
+# variables designed for the conda environents implimented in the MET-tools 
+# scripts.
+#
+##################################################################################
+# GLOBAL PARAMETERS TO BE SET BY USER
+##################################################################################
+
+# Root directory for MET-tools git clone
+export USR_HME=/home/jlconti/MET-tools
+
+# Refine the case-wise sub-directory for path names, leave as empty string if not needed
+export CSE=jlconti
+
+# Root directory for MET singularity image
+export SOFT_ROOT=/home/jlconti
+
+# Define first and last date time for forecast initialization (YYYYMMDDHH)
+export STRT_DT=2022123100
+export END_DT=2023011800
+
+# Array of control flow names to be processed
+CTR_FLWS=( 
+          "NRT_gfs"
+          "NRT_ecmwf"
+         )
+
+# Model grid / domain to be processed
+GRDS=( 
+      "d01"
+      "d02"
+      "d03"
+     )
+
+# Specify thresholds levels for verification
+export CAT_THR="[ >0.0, >=1.0, >=10.0, >=25.0, >=50.0 ]"
+
+# Define the interpolation method and related parameters
+INT_MTHDS=( 
+           "DW_MEAN"
+           "DW_MEAN"
+           "DW_MEAN"
+          )
+
+INT_WDTHS=( 
+           "3"
+           "9"
+           "27"
+          )
+
+# Singularity Container Variables for NetCDF Environment
+export NCKS_CMD="singularity exec --bind /cw3e:/cw3e,/scratch:/scratch /cw3e/mead/projects/cwp106/scratch/jlconti/MET_tools_conda_netcdf.sif ncks"
```

Comment:
Very small items to do cleanup with, I'm trying to keep the cwp106 scratch-level clone as an open example for folks to see and use as a reference.  If you can cleanup the references in the paths to point to this level, before the jlconti directory this would be a nice way to make this more user-agnostic.

Reply:
For more clarification, do you mean moving the path of the container to be relative to the user's home directory? Like say `/home/jlconti/containers...` vs the current `/cw3e/mead/projects/cwp106/...`

Reply:
I think making the paths relative to the directory
```
/cw3e/mead/projects/cwp106/scratch
```
would be perfect, assuming that the working example is at
```
/cw3e/mead/projects/cwp106/scratch/MET-tools
```
and related directories will be co-located at the scratch level.